src/services/PodManager.ts

```diff
@@ -77,7 +78,7 @@ export default class PodManager {
           `error=${e.message}`)
       }
 
-      this.pods.deletePod(pod)
+      await this.pods.deletePod(pod)
```

Comment:
Why not keep this asynchronous? I don't think there are any significant race conditions involving pod deletion.

Reply:
Felt like it was more consistent but it could be async since as you said there are no race conditions here. Ill move it back to being async.

Reply:
Sounds good. Just seemed like best practice in case the host needs to deal with a significant amount of reads/writes to the database.